### PR TITLE
[zh] Fix raw HTML data in the tutorials page

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Fixes: https://github.com/kubernetes/website/issues/44975

Synchronization is also performed.

See [the preview page](https://deploy-preview-45013--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro/)

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
/approve

**Comment by reylejano** [comment]
/retitle [zh] Fix raw HTML data in the tutorials page

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 1]
```diff
@@ -50,13 +50,14 @@ <h3>扩缩应用程序</h3>
...
+            <p>If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section</a>, you may have deleted the Service exposing the Deployment. In that case, please expose the Deployment again using the following command:</p><p><code><b>kubectl expose deployment/kubernetes-bootcamp --type="NodePort" --port 8080</b></code></p>
             -->
             <p><b>扩缩</b> 是通过改变 Deployment 中的副本数量来实现的。</p>
-            <p> <b> 注意 </b> 如果你是在 <a href="/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro/">上一节</a> 之后尝试此操作,
-              你可能需要从 <a href="/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">创建一个集群</a> 开始，
-              因为 Service 可能已被删除。</p>
-
+              {{< note >}}
+              <p>如果你是在 <a href="/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro/">上一节</a> 之后尝试此操作,
+                那么你可能已经删除了暴露 Deployment 的 Service。在这种情况下，请使用以下命令再次暴露 Deployment：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:58]: expose 这个词，翻译成“暴露”是直译，不过总是感觉有点儿负面的味道，是不是翻译成“公开”、“公布”更合适？

Comment (mengjiao-liu) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: 已更新。

**Pull request merged**